Extensions/ExtendedVariables.json

```diff
@@ -28,7 +37,8 @@
       "description": "Check if a global variable exists.",
       "fullName": "Global variable exists",
       "functionType": "Condition",
-      "name": "C_global_variable_exist",
+      "group": "Global variables",
+      "name": "GlobalVariableExist",
```

Comment:
Could we keep the old function make them private and call the new functions?

Reply:
I can add a warning message in the description to warn people.
Old functions will be shown as `Unknown`, so people will be forced to fix them.
I don't wanna duplicate all functions, the idea is to start again on a solid base.

Reply:
Without having a message that gives the name of the old function in the error message (like you suggested the other day), users won't know what was the function they used and more importantly its parameter expressions.

Old functions can just be copy-pasted and will be one liners. The private icon and strange name will make them easy to identify for extension maintainers. I think it's not a huge effort and it can save a lot of time to extension users.